server.js

```diff
@@ -26,10 +56,238 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.status(200).json({
+    Message: "Top-Music data",
+    data: listEndpoints(app)
+  });
+});
+
```

Comment:
Would suggest having a look at your routes order and see if perhaps /lists and /songsquery should be moved further up in your code? There was a section on this in week 13 common pitfalls, here's a screen shot. 
![image](https://user-images.githubusercontent.com/102091458/204740848-29c63e60-a969-4720-816c-be5c24a7f4e5.png)